pulsar/activity/publish/activity.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/project-flogo/core/support/log"
 )
 
-var logger = log.ChildLogger(log.RootLogger(), "pulsar-publish")
+var logger log.Logger
```

Comment:
Remove logger from global,  we don't need this to be global right?

Reply:
Agree. This is not needed.

Reply:
Changes Done